pkg/api/v2/remove.go

```diff
@@ -0,0 +1,24 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	snaputil "github.com/canonical/microk8s-cluster-agent/pkg/snap/util"
+)
+
+// RemoveRequest represents a request to remove a node from the dqlite cluster.
+type RemoveRequest struct {
+	// HostPort is the address of the node to remove.
+	HostPort string `json:"hostPort"`
+}
+
+// Remove implements the "POST /v2/remove" endpoint and removes a node from the dqlite cluster.
+func (a *API) Remove(ctx context.Context, req RemoveRequest) (int, error) {
```

Comment:
I'm not sure about the `/v2/remove` endpoint, this is not a full "remove" and more of a remove from the dqlite. What do you think of  `/v2/dqlite/remove`?